Dockerfile

```diff
@@ -1,6 +1,7 @@
 FROM quay.io/bitriseio/android-ndk:v2018_05_05-06_07-b990
 
-ENV TOOL_VER_BITRISE_CLI="1.48.0"
+ENV TOOL_VER_BITRISE_CLI="1.48.0" \
+    TOOL_VER_GO="1.16.5"
```

Comment:
don't we want a separate ENV TOOL_VER_GO="1.16.5" line here?

Reply:
we do the same way in bitrise-base Dockerfile, I tought that I would keep it like this 🤔

Reply:
it is readable in separate lines für mich, but I can accept it